config-linux.md

```diff
@@ -637,6 +637,30 @@ The following parameters can be specified to set up seccomp:
     "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
 ```
 
+### <a name="configLinuxRdmaLimits" />RDMA resource limits
+
+**`rdma_limits`** (array of objects, OPTIONAL) represents the `rdma` controller which allows the limiting of the
+resources related to network interfaces which support RDMA (e.g. HCA handles).
+For more information, see the RDMA section of [the kernel cgroups documentation][cgroup-v1-rdma].
+
+Each entry has the following structure:
+
+* **`interface_name`** *(string, REQUIRED)* - a name of an interface
+* **`hca_handle_limit`** *(int64, REQUIRED)* - limit a number of HCA handles which can be used by a container (-1 means max)
```

Comment:
Maybe follow [the kernel wording][1] and use:

    * **`hca_handle_limit`** (int64, REQUIRED) - Maximum number of HCA Handles (use `-1` to request the maximum available).

I think talking about “maximum number of” is less ambiguous than “limit a number of”, although I can't put my finger on why, so I'm also fine leaving this as it stands.

[1]: https://www.kernel.org/doc/Documentation/cgroup-v1/rdma.txt